lib/rust/ensogl/pack/src/lib.rs

```diff
@@ -25,29 +25,29 @@
 //!       │  ├─ runtime-libs.js          | Bundled `this_crate/js/runtime-libs`.
 //!       │  ╰─ snippets                 | Rust-extracted JS snippets.
 //!       │     ╰─ <name>.js             | A single Rust-extracted JS snippet.
-//!       ├─ shaders                     | Not optimized shaders sources extracted from WASM bundle.
-//!       │  ├─ list.txt                 | List of extracted not optimized shaders (no extensions).
-//!       │  ├─ <name>.<stage>.glsl      | A single not optimized shader. (Stage = vertex|fragment).
-//!       │  ╰─ ...
-//!       ├─ shaders-hash                | Not optimized shader hashes. Used to detect changes.
-//!       │  ├─ <name>.<stage>.hash      | A single not optimized shader hash.
-//!       │  ╰─ ...
+//!       ├─ dynamic-assets              | Dynamic asset sources extracted from WASM bundle.
+//!       │  ╰─ <type>                   | A subdirectory for each type of asset.
+//!       │     ├─ <key>                 | A subdirectory for each asset's sources.
+//!       │     ├─ <key>.work            | Intermediate files produced when building an asset.
```

Comment:
Could you please expand that to make it more clear? Just show their folders "shaders", and "glyph-cache" (or whatever name for the second folder is), and then, below them, show the `<type>`. This way, its easier to see examples of dynamic assets and how they are used for two example asset types.